certbot/docs/install.rst

```diff
@@ -70,11 +70,14 @@ The ``certbot-auto`` wrapper script installs Certbot, obtaining some dependencie
 from your web server OS and putting others in a python virtual environment. You can
 download and run it as follows::
 
-  user@webserver:~$ wget https://dl.eff.org/certbot-auto
-  user@webserver:~$ sudo mv certbot-auto /usr/local/bin/certbot-auto
-  user@webserver:~$ sudo chown root /usr/local/bin/certbot-auto
-  user@webserver:~$ chmod 0755 /usr/local/bin/certbot-auto
-  user@webserver:~$ /usr/local/bin/certbot-auto --help
+  cd /tmp
```

Comment:
I think this can probably be omitted. If we're worried about security, we could do something like `cd $(mktemp -d)`, but I think it's overkill and distracts people from the important parts of the instructions.

Reply:
it's just that you don't want to download the file in a random dir but then i realized that the file is moved anyway so yes it is quite useless